app/components/FFXIV/ItemDetailsTable.tsx

```
@@ -0,0 +1,145 @@
+      setSortDirection(sortDirection === 'asc' ? 'desc' : 'asc')
+    } else {
+      setSortColumn(columnId)
+      setSortDirection('asc')
+    }
+  }
+
+  const sortedData = [...data].sort((a, b) => {
+    const column = columnList.find((col) => col.columnId === sortColumn)
+    if (!column) return 0
+
+    let aValue: any = column.dataAccessor
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'aValue' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)